torchtext/csrc/vocab.cpp

```diff
@@ -264,44 +265,37 @@ _concat_tokens(std::vector<std::shared_ptr<IndexDict>> chunk_counters,
     unique_tokens.insert(unique_tokens.begin(), unk_token);
   }
 
-  // create stoi
-  IndexDict stoi;
-  stoi.reserve(num_lines);
-  int64_t index = 0;
-
-  for (const auto &token : unique_tokens) {
-    stoi[token] = index;
-    index++;
-  }
-
-  return std::make_tuple(std::move(stoi), std::move(unique_tokens));
+  return unique_tokens;
 }
 
-constexpr int64_t GRAIN_SIZE = 13107;
+constexpr uint32_t GRAIN_SIZE = 13107;
 Vocab _load_vocab_from_file(const std::string &file_path,
                             const std::string &unk_token,
-                            const int64_t min_freq, const int64_t num_cpus) {
+                            const uint32_t min_freq, const uint32_t num_cpus) {
```

Comment:
What's the purpose of changing these arguments from int64 to uint32?

Reply:
Will restore the original state, it's a result of trying out something else and unfortunately didn't get clean up properly.